small-talk/app/homepage/page.js

```diff
@@ -1,156 +1,71 @@
 
-    const hoverStyle = {
-        ...style,
-        ...(isHovered ? hoverEffect : null),
-    };
+    const hoverStyle = isHovered ? "button hover" : "button";
 
     return (
         <button
-            style={hoverStyle}
+            className={hoverStyle}
             onClick={onClick}
             onMouseEnter={() => setIsHovered(true)}
             onMouseLeave={() => setIsHovered(false)}
         >
-            {children}
+            <img src={iconImage} alt={altText} className="button-icon" />
         </button>
     );
 };
 
-
-const handleGame = () => {
-
-    window.location.href = "/game"; // Redirect to the game
-};
-
-const handleChat = () => {
-
-    window.location.href = "/chat"; // Redirect to the chat
-};
-
-const handleFood = () => {
-
-    window.location.href = "/food"; // Redirect to the food
-};
-
-const handleMovies = () => {
-
-    window.location.href = "/movie"; // Redirect to the movies
-};
-
-const handleSettings = () => {
-
-    window.location.href = "/setting"; // Redirect to the setting
-};
-
-const handleProfile = () => {
-
-    window.location.href = "/profile"; // Redirect to the profile
-};
-
-const handleSignOut = () => {
-
-    window.location.href = "/"; // Redirect to the welcome screen
-};
+const handleRedirect = (path) => () => {
+    window.location.href = path;
+}
 
 export default function Homepage() {
-
-
     return (
-        <div style={HomePageContainerStyle}>
+        <div className="homePageContainer">
             {/* header for homepage */}
-            <h1 style={HeaderStyle}>Welcome to Homepage</h1>
-            <div style={SidebarStyle}>
-                {/* creating a game button, and direct to the game page with ButtonStyling for the button */}
-                <HoverButton style={ButtonStyle} onClick={handleGame}>Game</HoverButton>
+            <h1 className="header">Welcome to Homepage</h1>
+            <div className = "topbar">
+                <img src="/img/three-bar.png" alt="Tab" style={{ height: '65px', marginLeft: '25px', marginTop: '40px' }}/>
+                <img src="/img/logo.png" alt="Logo" style={{ marginLeft: '45px', marginTop: '10px', marginBottom: '10px' }}/>
+                <button onClick={handleRedirect('/')}>
+                <img src="/img/logout-icon.png" alt="Notifications" style={{ height: '80px', marginTop: '45px', marginBottom: '10px', marginLeft: '1280px'}}/>
+                </button>
+                <button onClick={handleRedirect('/setting')}>
+                    <img src="/img/notif-icon.png" alt="Notifications" style={{ height: '80px', marginTop: '45px', marginBottom: '10px', marginLeft: '10px'}}/>
+                </button>
+                <button onClick={handleRedirect('/profile')}>
+                    <img src="/img/profile-temp.png" alt="Profile" style={{ height: '115px', marginTop: '10px', marginBottom: '10px', marginLeft: '20px' }}/>
+                </button>
+            </div>
```

Comment:
Currently shows in the vercel deployment:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/assets/80655017/ebcd4c99-fb11-434c-9d5d-13ee3cc2f644)


To help debug this, when I zoom out I see:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/assets/80655017/757213c2-acbc-4c7c-86e6-99f932f65578)

And when I zoom in, I see:
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/assets/80655017/96b62c05-60b9-4b9d-92f9-a1d06d814305)

Reply:
Looks good
![image](https://github.com/UNLV-CS472-672/2024-S-GROUP4-SMTK/assets/80655017/1a06cdf6-77b0-486a-97a0-c4ee4c36764c)